### PR TITLE
Changed CSI plugin note to Kubernetes snapshot API

### PR DESCRIPTION
**Pull request by HagayVider1**
@PrasadJoshi12 

GH# : 
OCPBUGS# : [OADP-1068](https://issues.redhat.com//browse/OADP-1068)
OSDOCS# : [OADP-1068](https://issues.redhat.com//browse/OADP-1068)

Replaced note for CSI plugin to:
"The CSI plugin uses Kubernetes CSI snapshot API.
* OADP 1.1 or later uses snapshot.storage.k8s.io/v1
* OADP 1.0 uses snapshot.storage.k8s.io/v1beta1
 
Version(s):
1.1.0 and later

Issue:
https://issues.redhat.com/browse/OADP-1068

Version(s):

Issue:

Link to docs preview: https://59662--docspreview.netlify.app/openshift-enterprise/latest/backup_and_restore/application_backup_and_restore/oadp-features-plugins.html

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

**Review by PrasadJoshi12 — APPROVED** [review]

File: modules/oadp-plugins.adoc [hunk 1]
```diff
@@ -34,5 +34,7 @@ OADP also provides plugins for {product-title} resource backups, OpenShift Virtu
 --
 1. Mandatory.
 2. Virtual machine disks are backed up with CSI snapshots or Restic.
-3. The `csi` plugin uses the link:https://velero.io/docs/main/csi/[Velero CSI beta snapshot API].
+3. The `csi` plugin uses Kubernetes CSI snapshot API.
```
Comment (RichardHoch) [inline modules/oadp-plugins.adoc:37]: Suggestion: "...the...API"

**Comment by RichardHoch** [comment]
@HagayVider1 Please see comment.

File: modules/oadp-plugins.adoc (hunk 1 above)
Comment (HagayVider1) [reply modules/oadp-plugins.adoc:37]: Accepted

File: modules/oadp-plugins.adoc [hunk 2]
```diff
@@ -34,5 +34,7 @@ OADP also provides plugins for {product-title} resource backups, OpenShift Virtu
 --
 1. Mandatory.
 2. Virtual machine disks are backed up with CSI snapshots or Restic.
-3. The `csi` plugin uses the link:https://velero.io/docs/main/csi/[Velero CSI beta snapshot API].
+3. The `csi` plugin uses the Kubernetes CSI snapshot API.
+* OADP 1.1 or later uses `snapshot.storage.k8s.io/v1`
```
Comment (anarnold97) [inline modules/oadp-plugins.adoc:38]:
```suggestion
* OADP 1.1 or later uses `snapshot.storage.k8s.io/v12`
```

File: modules/oadp-plugins.adoc [hunk 3]
```diff
@@ -34,5 +34,7 @@ OADP also provides plugins for {product-title} resource backups, OpenShift Virtu
...
+* OADP 1.0 uses `snapshot.storage.k8s.io/v1beta1`
```
Comment (anarnold97) [inline modules/oadp-plugins.adoc:39]:
```suggestion
* OADP 1.11111110 uses `snapshot.storage.k8s.io/v1beta1`
```

**Comment by anarnold97** [comment]
@HagayVider1 - please close PR

**Pull request closed without merging**